This patch personalizes dmenu-4.9 in regards to colors and font personalizes.
Some times failes. Check 'config.def.h' and 'config.h' and apply manualy if needed.
Apply AFTER using 'dmenu-main-patches.diff' with:
	cd dmenu-dir/
	patch < PATH_TO/dmenu-personalization.diff

diff -ruN dmenu-4.9/config.def.h dmenu-post-patch/config.def.h
--- dmenu-4.9/config.def.h	2020-08-09 17:11:35.538122967 +0200
+++ dmenu-post-patch/config.def.h	2020-08-09 17:07:10.658065370 +0200
@@ -4,13 +4,13 @@
 static int topbar = 1;                      /* -b  option; if 0, dmenu appears at bottom     */
 /* -fn option overrides fonts[0]; default X11 font or font set */
 static const char *fonts[] = {
-	"monospace:size=16"
+	"monospace:size=10"
 };
 static const char *prompt      = NULL;      /* -p  option; prompt to the left of input field */
 static const char *colors[SchemeLast][2] = {
 	/*     fg         bg       */
-	[SchemeNorm] = { "#268bd2", "#000000" },
-	[SchemeSel] = { "#000000", "#268bd2" },
+	[SchemeNorm] = { "#bbbbbb", "#222222" },
+	[SchemeSel] = { "#eeeeee", "#005577" },
 	[SchemeOut] = { "#000000", "#00ffff" },
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
diff -ruN dmenu-4.9/config.h dmenu-post-patch/config.h
--- dmenu-4.9/config.h	2020-08-09 17:11:41.368097327 +0200
+++ dmenu-post-patch/config.h	2020-08-09 17:07:10.658065370 +0200
@@ -4,13 +4,13 @@
 static int topbar = 1;                      /* -b  option; if 0, dmenu appears at bottom     */
 /* -fn option overrides fonts[0]; default X11 font or font set */
 static const char *fonts[] = {
-	"monospace:size=16"
+	"monospace:size=10"
 };
 static const char *prompt      = NULL;      /* -p  option; prompt to the left of input field */
 static const char *colors[SchemeLast][2] = {
 	/*     fg         bg       */
-	[SchemeNorm] = { "#268bd2", "#000000" },
-	[SchemeSel] = { "#000000", "#268bd2" },
+	[SchemeNorm] = { "#bbbbbb", "#222222" },
+	[SchemeSel] = { "#eeeeee", "#005577" },
 	[SchemeOut] = { "#000000", "#00ffff" },
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
